web/packages/design/src/ResourceIcon/index.tsx

```diff
@@ -45,7 +45,9 @@ export const ResourceIcon = ({ name, ...props }: ResourceIconProps) => {
   if (!icon) {
     return null;
   }
-  return <Image src={icon} {...props} />;
+  // Note: we add the class name for consistency with `Icon`, where it's used
+  // for testing.
+  return <Image src={icon} className={`res-icon-${name}`} {...props} />;
```

Comment:
i wrote that test when i was just starting out 😅, can we use data-testid instead?